pill-press-app/ClientApp/src/app/equipment-notification/equipment-location/equipment-location.component.ts

```diff
@@ -59,6 +59,112 @@ export class EquipmentLocationComponent extends FormBase implements OnInit {
   ngOnInit() {
     this.form = this.fb.group({
       id: [],
+      ownedBeforeJan2019: [],
```

Comment:
Is there a reason why all these fields are being added to the formGroup? This page only manipulates the equipment location.

Reply:
This was done as part of work to get the form to function.  It may be redundant.

Reply:
Approving temporarily for users to test the LSD changes